fourth.py

```diff
@@ -0,0 +1,11 @@
+import time, sys, keyboard
+
+target = int(sys.argv[1])
```

Comment:
use argparse for arguments, use argument with flag. Example python3 main.py --target 10. where target is a cli flag.